src/main/java/mate/jdbc/dao/DriverDaoImpl.java

```diff
@@ -0,0 +1,116 @@
+package mate.jdbc.dao;
+
+import java.sql.Connection;
+import java.sql.PreparedStatement;
+import java.sql.ResultSet;
+import java.sql.SQLException;
+import java.sql.Statement;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Optional;
+import mate.jdbc.exception.DataProcessingException;
+import mate.jdbc.lib.Dao;
+import mate.jdbc.model.Driver;
+import mate.jdbc.util.ConnectionUtil;
+
+@Dao
+public class DriverDaoImpl implements DriverDao {
+    @Override
+    public Driver create(Driver driver) {
+        String query = "INSERT INTO drivers (name, licenseNumber) "
+                    + "VALUES (?, ?)";
+        try (Connection connection = ConnectionUtil.getConnection();
+                PreparedStatement statement
+                         = connection.prepareStatement(query, Statement.RETURN_GENERATED_KEYS)) {
+            statement.setString(1, driver.getName());
+            statement.setString(2, driver.getLicenseNumber()); // Q/A about parameter index.
+            statement.executeUpdate();
+            ResultSet resultSet = statement.getGeneratedKeys();
+            if (resultSet.next()) {
+                driver.setId(resultSet.getObject(1, Long.class));
+            }
+            return driver;
+        } catch (SQLException e) {
+            throw new DataProcessingException("Couldn't create driver. " + driver, e);
+        }
+    }
+
+    @Override
+    public Optional<Driver> get(Long id) {
+        String query = "SELECT * FROM drivers"
+                + " WHERE id = ? AND is_deleted = FALSE";
+        try (Connection connection = ConnectionUtil.getConnection();
+                 PreparedStatement statement = connection.prepareStatement(query)) {
+            statement.setLong(1, id);
+            ResultSet resultSet = statement.executeQuery();
+            Driver driver;
+            if (resultSet.next()) {
+                driver = getDriver(resultSet);
+                return Optional.of(driver); // google about ofNullable method
```

Comment:
```suggestion
                return Optional.of(driver); 
```
check the source code, this method does a pretty easy job

Reply:
> check the source code, this method does a pretty easy job

checked)